docs/flows.md

```diff
@@ -137,27 +140,42 @@ To deploy flows using Cloud Run and similar services, define your flows using `d
 ```js
 import { defineFlow, startFlowsServer } from '@genkit-ai/flow';
 
-export const jokeFlow = defineFlow({ name: 'jokeFlow' }, async (subject) => {
-  // ....
-});
+export const menuSuggestionFlow = defineFlow(
+  {
+    name: 'menuSuggestionFlow',
+  },
+  async (cuisine) => {
+    // ....
+  }
+);
 
 startFlowsServer();
 ```
 
-By default `startFlowsServer` will serve all the flows that you have defined in your codebase as HTTP endpoints (e.g. `http://localhost:3400/jokeFlow`).
+By default `startFlowsServer` will serve all the flows that you have defined in your codebase as HTTP endpoints (e.g. `http://localhost:3400/menuSuggestionFlow`).
 
 You can choose which flows are exposed via the flows server. You can specify a custom port (it will use the `PORT` environment variable if set). You can also set CORS settings.
 
 ```js
 import { defineFlow, startFlowsServer } from '@genkit-ai/flow';
 
-export const flowA = defineFlow({ name: 'flowA' }, async (subject) => {
-  // ....
-});
+export const flowA = defineFlow(
+  {
+    name: 'menuSuggestionFlowA',
```

Comment:
`flowA` / `flowB` works because its generic in nature. I am not sure `menuSuggestionFlowA` / `menuSuggestionFlowB` reads the same way. Should we stick with flowA/flowB? or introduce a second flow name into the mix?

Reply:
Went back to `flowA` and `flowB`.